hazelcast/src/test/java/com/hazelcast/config/XmlConfigImportVariableReplacementTest.java

```diff
@@ -411,7 +411,7 @@ public void testImportOverlappingMapConfigFromFile() throws Exception {
         // YAML recursively merges the two files
         assertEquals(1, myMapConfig.getBackupCount());
         MapStoreConfig myMapStoreConfig = myMapConfig.getMapStoreConfig();
-        assertEquals(0, myMapStoreConfig.getWriteDelaySeconds());
+        assertEquals(10, myMapStoreConfig.getWriteDelaySeconds());
```

Comment:
This was broken already. At the beginning of the test, `10` is set, but the assertion would verify the value `0` which is the default one.